README.md

```diff
@@ -46,25 +46,33 @@ Citus Cloud runs on top of AWS as a fully managed database as a service and has
 
 If you're looking to get started locally, you can follow the following steps to get up and running.
 
-* Install docker-compose: [Mac][mac_install] | [Linux][linux_install]
-* (Mac only) connect to Docker VM
-  ```bash
-  eval $(docker-machine env default)
-  ```
-
-* Pull and start the docker images
+1. Install Docker Community Edition and docker-compose
+  * Mac:
+    1. [Download](https://www.docker.com/community-edition#/download) and install Docker.
+    2. Start Docker by clicking on the application’s icon.
+  * Linux:
+    ```bash
+    curl -sSL https://get.docker.com/ | sudo bash
+    sudo usermod -aG docker $USER && newgrp docker
+    sudo systemctl start docker
+
+    sudo curl -L https://github.com/docker/compose/releases/download/1.11.2/docker-compose-`uname -s`-`uname -m` -o /usr/local/bin/docker-compose
```

Comment:
Instead of all the `uname` parts here, what if we just hardcode _Linux-x86_64_? There are no other Linux variants, so it doesn't seem worth the trouble of all the backticks/subprocesses.

On the other hand, I suppose if it produces a URL that doesn't exist, the user will see an error, so I could go either way.